cromwell-backend/src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -0,0 +1,348 @@
+package cromwell.backend.provider.local
+
+import java.nio.file.{Files, Path, Paths}
+import java.util.regex.Pattern
+
+import akka.actor.{ActorRef, Props}
+import akka.util.Timeout
+import better.files._
+import com.typesafe.scalalogging.StrictLogging
+import cromwell.backend.BackendActor
+import cromwell.backend.model._
+import cromwell.backend.provider.local.FileExtensions._
+import wdl4s.WdlExpression
+import wdl4s.types.{WdlFileType, WdlType}
+import wdl4s.values.{WdlFile, WdlSingleFile, WdlValue}
+
+import scala.annotation.tailrec
+import scala.collection.mutable.ArrayBuffer
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.sys.process._
+import scala.util.{Random, Try}
+
+object LocalBackend {
+  // Folders
+  val CromwellExecutionDir = "cromwell-executions"
```

Comment:
Yeap, you are right... but we were thinking on a more complex configuration implementation due to the use of multiple jars that may overwrite configuration... so for that we will extending BackendConfiguration implementation.
That's why you see hardcoded configuration right now.